tests/nl/_common.yaml

```diff
@@ -21,3 +21,7 @@ entities:
     id: fan.plafond
     area: woonkamer
     domain: fan
+  - name: Het Stofje
```

Comment:
Not happy with this. It's a stupid name. However, we have a bunch of names hardcoded that include common names. Couldn't think of anything better.